content/en/tracing/profiling/_index.md

```diff
@@ -54,20 +54,26 @@ The Datadog Profiler requires [Java Flight Recorder][1]. The Datadog Profiling l
 
 - Because profiles are sent directly to Datadog without using the Datadog Agent, you must pass a valid [Datadog API key][4].
 
-- As an alternative to passing arguments, you can use environment variable to set those parameters:
+- For advanced setup of the profiler or to add tags like `service` or `version`, use environment variables to set the parameters:
 
-| Arguments                     | Environment variable      | Description                                       |
-| ----------------------------- | ------------------------- | ------------------------------------------------- |
-| `-Ddd.profiling.enabled`      | DD_PROFILING_ENABLED      | Set to `true` to enable profiling.                |
-| `-Ddd.profiling.api-key-file` | DD_PROFILING_API_KEY_FILE | File that should contain the API key as a string. |
-|                               | DD_PROFILING_API_KEY      | Datadog API key.                                  |
-| `-Ddd.site`                   | DD_SITE                   | Destination site for your profiles (versions 0.48+). Valid options are `datadoghq.com` for Datadog US site (default), and `datadoghq.eu` for the Datadog EU site. |
+| Environment variable                             | Type          | Description                                                                                      |
+| ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
+| `DD_PROFILING_ENABLED`                           | Boolean       | Alternate for `-Ddd.profiling.enabled` argument. Set to `true` to enable profiling.               |
+| `DD_API_KEY`                                     | String        | Alternate for `-Ddd.profiling.api-key-file` argument. The [Datadog API key][4] to use when uploading profiles. |
+| `DD_PROFILING_API_KEY_FILE`                      | String        | Replacement for `-Ddd.profiling.api-key-file` argument. File that should contain the [Datadog API key][4].  |
+| `DD_SITE`                                        | String        | Destination site for your profiles (versions 0.48+). Valid options are `datadoghq.com` for Datadog US site (default), and `datadoghq.eu` for the Datadog EU site. |
+| `DD_SERVICE`                                     | String        | The Datadog [service][5] name.     |
+| `DD_ENV`                                         | String        | The Datadog [environment][6] name, for example `production`.|
+| `DD_VERSION`                                     | String        | The version of your application.                             |
+| `DD_TAGS`                                        | String        | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas such as: `layer:api, team:intake`.  |
```

Comment:
Ooh, I like how you got rid of the "Arguments" col!

Any reason not to include the command line version for all of these?

Reply:
I added them in the notes section. Do we support the others as well as cli arguments? ie `-Ddd.tags="<key>:<val>"` etc